browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -232,6 +232,67 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_RESOLVE_METHOD_ETHEREUM" desc="Text for resolve method Ethereum.">
         Ethereum
       </message>
+      <message name="IDS_BRAVE_NEWS_OPTIN_TITLE" desc="Text for opt in card title">
```

Comment:
Can these strings be shared between desktop and android? The UI is almost all the same.